models/users.js

```diff
@@ -154,5 +158,58 @@ module.exports = {
     }
     const res = await db.query(auth)
     return (res.rows.length !== 0)
+  },
+
+  getMergedUserId: async function (userId) {
+    const getMergeUserId = {
+      text: 'SELECT mergeduserid FROM users WHERE id = $1',
+      values: [userId]
+    }
+    const res = await db.query(getMergeUserId)
+    return res.rows[0].mergeduserid
+  },
```

Comment:
Need to return null if user is missing.

Since we're duplicating this validation, maybe we should write a helper function?

Reply:
The mergeuserid column is by default null. 
Do you mean something like getAllMergeUserIds?

Reply:
What happens if the `userId` is not found in the database at all? Other UserModel functions have an if-statement that checks if the query was successful before returning.

Reply:
Ahh ok I see what you mean.